ogc/src/main/scala/geotrellis/server/ogc/style/ColorRampStyle.scala

```diff
@@ -59,21 +59,22 @@ case class ColorRampStyle(
     // The number of stops between each provided break
     val numStops: Int = stops.getOrElse(colorRamp.colors.length)
 
-    // The colors, interpolated (invisible added for values below our minimum break)
-    val interpolatedColors: Vector[Int] =
-      0x00000000 +: colorRamp.stops(numStops)
+    // The colors, interpolated (colors added at start and end for out of bounds values)
+    val minColor = if (clampWithColor) colorRamp.colors.head else 0x00000000
+    val maxColor = if (clampWithColor) colorRamp.colors.last else 0x00000000
```

Comment:
Why should the maxColor be set to `0x00000000`?

Reply:
The way I'm reading it is if the rendering is not clamped to max color value, that is forced to use it, it should be transparent.